api/src/routes/team.js

```diff
@@ -61,6 +61,14 @@ teamRouter.delete(
   teamController.deleteTeamMember,
 );
 
+teamRouter.patch(
+  '/:teamId/members',
```

Comment:
if the refactor of the patch route is trivial, we could do that too... up to you

Reply:
I didn't know we should use plural actually.... I use "members" instead of "member" as it will conflict with the updateTeamMember endpoint.
Cause both of them are POST request with endpoint of /team/${teamId}/member (updateTeamMember ) /team/${teamId}/members(deleteBatchTeamMembers), any suggestion on re-naming the deleteBatchTeamMembers endpoints?

Reply:
Can we push the refactoring to a later PR and maybe discuss the approach in winter's meeting?

Reply:
Ahh woops my bad I wasn’t clear. In my second comment on this thread, I meant that the route for update team member is not configured as intended: rather than /:teamId/member it should be /:teamId/members/:memberId, like we are doing for the delete. If that’s renamed, you won’t have a conflict

Reply:
@aksh0001 Will create a separate ticket to refractor it (if we have time)